src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -330,36 +375,41 @@
 						<FontIcon Glyph="{x:Bind Commands.InstallInfDriver.Glyph.BaseGlyph}" />
 					</AppBarButton.Icon>
 				</AppBarButton>
+
+				<!--  Rotate Image Left  -->
 				<AppBarButton
 					x:Name="RotateImageLeftButton"
 					x:Load="{x:Bind ViewModel.IsImage, Mode=OneWay, FallbackValue=False}"
 					Command="{x:Bind Commands.RotateLeft, Mode=OneWay}"
 					Label="{x:Bind Commands.RotateLeft.Label}"
 					LabelPosition="Default"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=RotateLeft}">
-
 					<local:OpacityIcon Style="{StaticResource ColorIconRotateLeft}" />
 				</AppBarButton>
+
+				<!--  Rotate Image Right  -->
 				<AppBarButton
 					x:Name="RotateImageRightButton"
 					x:Load="{x:Bind ViewModel.IsImage, Mode=OneWay, FallbackValue=False}"
 					Command="{x:Bind Commands.RotateRight, Mode=OneWay}"
 					Label="{x:Bind Commands.RotateRight.Label}"
 					LabelPosition="Default"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=RotateRight}">
-
 					<local:OpacityIcon Style="{StaticResource ColorIconRotateRight}" />
 				</AppBarButton>
+
+				<!--  Install Font  -->
 				<AppBarButton
 					x:Name="InstallFontButton"
 					x:Load="{x:Bind Commands.InstallFont.IsExecutable, Mode=OneWay, FallbackValue=False}"
 					Command="{x:Bind Commands.InstallFont, Mode=OneWay}"
 					Label="{x:Bind Commands.InstallFont.Label}"
 					LabelPosition="Default"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=Install}">
-
-					<!--  Add icon  -->
+					<!--  TODO: Add Icon  -->
```

Comment:
TODO?

Reply:
If it is not todo, what's supposed to be meant 'Add icon'?

Reply:
@yaira2

Reply:
It's a todo for myself to add an opacity icon.